README.md

```diff
@@ -4,130 +4,182 @@
+You can incrementally migrate your Python code over (or continue to use your
+favorite Python libraries), because you can easily call your favorite Python
+library with a familiar syntax:
 
-## Installation and Usage
+```swift
+import TensorFlow
+import Python
 
-You can [download a pre-built package](Installation.md) for Swift for TensorFlow. 
-After installation, you can follow these [instructions](Usage.md) to try it out.
+let np = Python.import("numpy")
 
-For instructions on building from source, visit
-[apple/swift](https://github.com/apple/swift/tree/tensorflow).
+let array = np.arange([10, 10])  // Create a 10x10 numpy array.
+let tensor = Tensor(numpy: array)  // Seamless integration!
+```
 
 ## Documentation
 
-Below are some documents explaining the Swift for TensorFlow project.
-
-### Conceptual overview
+### Getting started
 
 - [Swift for TensorFlow Design Overview](docs/DesignOverview.md)
 - [Why *Swift* for TensorFlow?](docs/WhySwiftForTensorFlow.md)
+- [Sample Models](https://github.com/tensorflow/swift-models)
+- [Tutorials](https://github.com/tensorflow/swift-tutorials)
 - [Frequently Asked Questions](FAQ.md)
+- [Swift Tensor API Reference](https://www.tensorflow.org/api_docs/swift/Structs/Tensor)
 
 ### Technology deep dive
 
-- [Graph Program Extraction](docs/GraphProgramExtraction.md)
+The Swift for TensorFlow project builds on top of powerful theoretical
+foundations. For insight into some of the underlying technologies, check
+out the following documentation.
+
+> Beware: the project is moving very quickly, and thus some of these documents
+> are slightly out of date as compared to the current state-of-the-art.
+
 - [Automatic Differentiation Whitepaper](docs/AutomaticDifferentiation.md)
 - [Automatic Differentiation Manifesto](https://gist.github.com/rxwei/30ba75ce092ab3b0dce4bde1fc2c9f1d)
 - [Python Interoperability](docs/PythonInteroperability.md)
 - [Aggregate Parameters and Parameter Update](docs/AggregateParameters.md)
-
-### Swift API reference
-
-- [TensorFlow](https://www.tensorflow.org/api_docs/swift/Structs/Tensor)
-
-### Design proposals
-
-- [One Graph Function per Host Function](proposals/OneGraphFunctionPerHostFunction.md)
+- [Graph Program Extraction](docs/GraphProgramExtraction.md)
 
 ## Source code
 
-The active development of Swift for TensorFlow will happen under the 
-"tensorflow" branch of
-[apple/swift](https://github.com/apple/swift/tree/tensorflow).
-
-These projects include:
+Compiler and standard library development happens on the `tensorflow` branch of
+the [apple/swift](https://github.com/apple/swift/tree/tensorflow) repository.
 
-- The compiler and standard libraries: [apple/swift](http://github.com/apple/swift/tree/tensorflow)
-- Debugger and REPL support: [apple/swift-lldb](http://github.com/apple/swift-lldb/tree/tensorflow)
+Additional code repositories that make up the core of project include:
```

Comment:
Small typo I found. Excited about the progress! 😄 
```suggestion
Additional code repositories that make up the core of the project include:
```

Reply:
Thanks for the catch! Fixed up.